config/test.json5

```diff
@@ -3,5 +3,23 @@
   blockchain: {
     provider: 'ws://localhost:8545',
     waitBlockCountBeforeConfirmationRemoved: 10
+  },
+  rns: {
+    enabled: true,
+    owner: {
+      contractAddress: "OWNER_ADDR"
```

Comment:
What is the reason for these placeholders? Failing tests?

Reply:
Right, we needed the addresses defined in the `test.json5` file so they can be picked in the Unit tests, because they are actually used in the Even HAndlers.

Reply:
Is this OK? Do you see a better way to mockup these config file values for the Tests?